.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Prerequisites (Remove this section if you want)
+Make sure you...
+
+* Follow the [contribution guidelines](CONTRIBUTING.md)
+* Follow the code style (run `./gradlew checkstyleMain`)
+* Submit this PR against the `Development` branch.
+
+**Fixed issue:** [Reference the issue number here, or remove if not a fix]
```

Comment:
Does this trigger the link PR -> Issue as well? (when you literally say "Closes #xxx, Fixes #xxx, ..." it will automatically close that issue when the PR is merged)